### PR TITLE
14731: CVUT coverage: Implement the --no-plugin flag of Power BI Visuals Tools (PBIVIZ)

### PR DESCRIPTION
**Pull request by ignatvilesov**
* 14731: CVUT coverage: Implement the --no-plugin flag of Power BI Visuals Tools (PBIVIZ)

### CODE REVIEW

**Comment by kamilzakiev** [comment]
Approved.

File: lib/TypescriptCompiler.js [hunk 1]
```diff
@@ -227,15 +227,6 @@ function concatExternalJS(visualPackage) {
     });
 }
 
-function createTypingsLink(visualPackage) {
```
Comment (AviSander) [inline lib/TypescriptCompiler.js]: why is this removed?

Comment (ignatvilesov) [inline lib/TypescriptCompiler.js]: I haven't found any usage, so, I decided to remove it in order to keep the clean code.

File: lib/TypescriptCompiler.js [hunk 2]
```diff
@@ -245,17 +236,24 @@ class TypescriptCompiler {
...
             //TODO: visual specific TS validation
             let tsconfig = require(visualPackage.buildPath('tsconfig.json'));
-            let pluginPath = createPlugin(visualPackage, pluginName);
             let tmpFiles = copyPackageFilesWithNamespace(visualPackage, tsconfig.files);
             let visualJsPath = visualPackage.buildPath(config.build.dropFolder, config.build.js);
-            //createTypingsLink(visualPackage);
-            tmpFiles.push(pluginPath);
+
+            if (options.plugin) {
+                const pluginPath = createPlugin(visualPackage, pluginName);
```
Comment (AviSander) [inline lib/TypescriptCompiler.js]: ConsoleWriter to log this?

Comment (ignatvilesov) [inline lib/TypescriptCompiler.js]: Done.
